### ISSUE TITLE
Broken class refrence in conversion test model Magnetic.FundamentalWave.Issue3075

### ISSUE THREAD

**Issue by dietmarw** [modelica/ModelicaStandardLibrary#3380]
When working on #3379 I found a broken reference. 
@christiankral  Once #3379 is merged I suggest you have a look on how to fix that.

Error message:
```
Checking model ModelicaTestConversion4.Magnetic.FundamentalWave.Issue3075:

Use of undeclared variable smee.stator.strayReluctance.abs_V_m
In class ModelicaTestConversion4.Magnetic.FundamentalWave.Issue3075.
File: /home/dietmarw/.workspace/gh-modelica/Modelica/ModelicaTestConversion4.mo, line 1427
Used in component V_m1.
File: /home/dietmarw/.workspace/gh-modelica/Modelica/ModelicaTestConversion4.mo, line 1428

In value attribute for V_m1.
Modifier applied in class ModelicaTestConversion4.Magnetic.FundamentalWave.Issue3075.
File: /home/dietmarw/.workspace/gh-modelica/Modelica/ModelicaTestConversion4.mo, line 1427
Modifier applied for component V_m1.
File: /home/dietmarw/.workspace/gh-modelica/Modelica/ModelicaTestConversion4.mo, line 1428
Variable found in class ModelicaTestConversion4.Magnetic.FundamentalWave.Issue3075.
Near file: /home/dietmarw/.workspace/gh-modelica/Modelica/ModelicaTestConversion4.mo, line 1427

Use of undeclared variable smee.excitation.reluctance.abs_V_m
In class ModelicaTestConversion4.Magnetic.FundamentalWave.Issue3075.
File: /home/dietmarw/.workspace/gh-modelica/Modelica/ModelicaTestConversion4.mo, line 1427
Used in component V_m2.
File: /home/dietmarw/.workspace/gh-modelica/Modelica/ModelicaTestConversion4.mo, line 1429

In value attribute for V_m2.
Modifier applied in class ModelicaTestConversion4.Magnetic.FundamentalWave.Issue3075.
File: /home/dietmarw/.workspace/gh-modelica/Modelica/ModelicaTestConversion4.mo, line 1427
Modifier applied for component V_m2.
File: /home/dietmarw/.workspace/gh-modelica/Modelica/ModelicaTestConversion4.mo, line 1429
Variable found in class ModelicaTestConversion4.Magnetic.FundamentalWave.Issue3075.
Near file: /home/dietmarw/.workspace/gh-modelica/Modelica/ModelicaTestConversion4.mo, line 1427

Use of undeclared variable smeeQS.stator.strayReluctance.abs_V_m
In class ModelicaTestConversion4.Magnetic.FundamentalWave.Issue3075.
File: /home/dietmarw/.workspace/gh-modelica/Modelica/ModelicaTestConversion4.mo, line 1427
Used in component V_m3.
File: /home/dietmarw/.workspace/gh-modelica/Modelica/ModelicaTestConversion4.mo, line 1430

In value attribute for V_m3.
Modifier applied in class ModelicaTestConversion4.Magnetic.FundamentalWave.Issue3075.
File: /home/dietmarw/.workspace/gh-modelica/Modelica/ModelicaTestConversion4.mo, line 1427
Modifier applied for component V_m3.
File: /home/dietmarw/.workspace/gh-modelica/Modelica/ModelicaTestConversion4.mo, line 1430
Variable found in class ModelicaTestConversion4.Magnetic.FundamentalWave.Issue3075.
Near file: /home/dietmarw/.workspace/gh-modelica/Modelica/ModelicaTestConversion4.mo, line 1427
```

**Comment by beutlich** [comment]
Duplicate of #3336.
